pmd-plsql/etc/grammar/PldocAST.jjt

```diff
@@ -1809,11 +1809,8 @@ void FromClauseEntry() #void :
 {}
 {
  (
-     LOOKAHEAD(JoinClause()) JoinClause()
- |
-     // using lookahead to avoid misinterpreting the token after the identifier
-     // as a table alias rather than as a keyword
-     LOOKAHEAD(<IDENTIFIER> (<PARTITION>|<NATURAL>|<OUTER>|<JOIN>|<RIGHT>|<CROSS>|<FULL>)) JoinClause()
+     LOOKAHEAD([ LOOKAHEAD(2) SchemaName() "." ] TableName()
```

Comment:
Looks like the fix is a better lookahead. We should make sure to have these edge cases covered:
*  a join with a table specified by "schema.tablename" instead of simple "tablename" . I'm not sure, whether this `[<IDENTIFIER>]` (see next line) is still needed.
* left join

Reply:
ok, `[<IDENTIFIER>]` is the table alias